src/pobox.erl

```diff
@@ -337,3 +510,59 @@ pop(stack, []) -> {empty, []};
 pop(stack, [H|T]) -> {{value,H}, T};
 pop(keep_old, Q) -> queue:out(Q);
 pop({mod, Mod}, Data) -> Mod:pop(Data).
+
+
+send_ownership_transfer(_PreviousOwnerPid, undefined, _HeirData, _BoxName, _Reason) -> {error, noproc};
+send_ownership_transfer(PreviousOwnerPid, NewOwnerName, HeirData, BoxName, Reason)
+    when ?PROCESS_NAME_GUARD_WITH_LOCAL_NO_PID(NewOwnerName), ?PROCESS_NAME_GUARD(BoxName) ->
+    case where(NewOwnerName) of
+        Pid when is_pid(Pid) ->
+            Pid ! {pobox_transfer, self(), PreviousOwnerPid, HeirData, Reason},
+            {ok, Pid};
+        _ ->
+            {error, noproc}
+    end;
+send_ownership_transfer(PreviousOwnerPid, NewOwnerPid, HeirData, BoxName, Reason)
+    when is_pid(NewOwnerPid), ?PROCESS_NAME_GUARD(BoxName) ->
+    NewOwnerPid ! {pobox_transfer, self(), PreviousOwnerPid, HeirData, Reason},
+    {ok, NewOwnerPid}.
+
+where(Pid) when is_pid(Pid) -> Pid;
+where(Name) when is_atom(Name) -> erlang:whereis(Name);
+where({global, Name}) -> global:whereis_name(Name);
+where({via, Module, Name}) -> Module:whereis_name(Name);
+where({local, Name}) when is_atom(Name) -> erlang:whereis(Name).
```

Comment:
that last form is never really used outside of when registering a process. I'd avoid supporting it.

Reply:
Done.